src/otx/core/model/entity/base.py

```diff
@@ -106,3 +109,62 @@ def map_class_names(src_classes: list[str], dst_classes: list[str]) -> list[int]
             else:
                 src2dst.append(-1)
         return src2dst
+
+
+class OVModel(OTXModel, Generic[T_OTXBatchDataEntity, T_OTXBatchPredEntity]):
+    """Base class for the OpenVINO model."""
+
+    def __init__(self, config: DictConfig) -> None:
+        self.model_name = config.pop("model_name")
+        self.model_type = config.pop("model_type")
+        self.async_inference = config.pop("async_inference", False)
+        self.num_requests = config.pop("max_num_requests", get_default_async_reqs_num())
+        self.use_tp_mode = config.pop("use_tp_mode", False)
```

Comment:
For me, `use_tp_mode` is unreadable. How about `use_throughput_mode`? It's a little bit long, though.

Reply:
Ok

Reply:
[done](https://github.com/openvinotoolkit/training_extensions/pull/2769/commits/1da85964b12868bb793297fab408ea395fe88500)